ci/build.bat

```diff
@@ -42,19 +48,19 @@ where.exe neovim-node-host.cmd || goto :error
 
 mkdir .deps
 cd .deps
-cmake -G "MinGW Makefiles" -DCMAKE_BUILD_TYPE=RelWithDebInfo ..\third-party\ || goto :error
-mingw32-make VERBOSE=1 || goto :error
+cmake -G %CMAKE_GENERATOR% -DCMAKE_BUILD_TYPE=RelWithDebInfo ..\third-party\ || goto :error
+cmake --build . -- %CMAKE_GENERATOR_ARGS% || goto :error
 cd ..
 
 :: Build Neovim
 mkdir build
 cd build
-cmake -G "MinGW Makefiles" -DCMAKE_BUILD_TYPE=RelWithDebInfo -DBUSTED_OUTPUT_TYPE=nvim %USE_GCOV% -DGPERF_PRG="C:\msys64\usr\bin\gperf.exe" .. || goto :error
-mingw32-make VERBOSE=1 || goto :error
+cmake -G %CMAKE_GENERATOR% -DCMAKE_BUILD_TYPE=RelWithDebInfo -DBUSTED_OUTPUT_TYPE=nvim %USE_GCOV% -DGPERF_PRG="C:\msys64\usr\bin\gperf.exe" .. || goto :error
```

Comment:
Looks like `C:\msys64\usr\bin\gperf.exe` is present on the default AppVeyor image.

Reply:
Yes, it is on the AppVeyor image, but I was wondering if we could add it to the [wintools.zip](https://github.com/neovim/deps/blob/master/opt/win32tools.zip) so Windows devs wouldn't need to download it themselves. Then the only requirements for building Neovim on Windows would be Visual Studio 2017, CMake, and git.

Reply:
wintools.zip is for end-users, but we could instead host gperf.exe in the [deps repo](https://github.com/neovim/deps/tree/master/opt) and automatically download it to `.deps` with CMake.